src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx

```diff
@@ -132,11 +134,10 @@ export class ConnectedApps extends Component {
             <AlertBox
               key={`${app.attributes?.title}`}
               className="vads-u-margin-bottom--2"
-              headline={`We couldn’t disconnect ${app.attributes?.title}`}
-              status="error"
-              content={`We’re sorry. Something went wrong on our end and we couldn’t disconnect ${
+              status={ALERT_TYPE.ERROR}
+              content={`We’re sorry. We can’t disconnect this app from your VA.gov profile right now: ${
                 app.attributes?.title
-              }. Please try again later.`}
+              }. We're working to fix this problem. Please check back later.`}
```

Comment:
one more here :( We have an ESLint rule for this kind of thing but I don't know why it's not working :(